notebook/tests/selenium/utils.py

```diff
@@ -100,6 +100,20 @@ def focus_cell(self, index=0):
         self.to_command_mode()
         self.current_cell = cell
 
+    def find_and_replace(self, index=0, find_txt='', replace_txt='', replace_all=False):
+        self.focus_cell(index)
+        esc(self.browser, 'F')
+        time.sleep(1)    # TODO: find better way to fill the find and replace form
+        self.browser.find_elements_by_css_selector("button.btn.btn-default.btn-sm")[2].click()
```

Comment:
What's this doing?

Reply:
This code is basically for clicking double "upside down" arrow (third button) on the find and replace form to enable global replace. It's kinda hacky way of doing it, since the buttons don't have any specific ids on them which made it harder to fetch them directly. But let me will revisit this commit to see if we can make it more robust. Thanks !

Reply:
We could add an ID, so long as that's not going to cause problems. If you do, try closing and reopening the dialog a few times so it's created several successive buttons with the same ID, and check that the button still works.